core/common/TimeUtil.cpp

```diff
@@ -404,4 +405,22 @@ std::string NumberToDigitString(uint32_t number, uint8_t length) {
     return result;
 }
 
+std::chrono::nanoseconds GetTimeDiffFromBoot() {
+#if defined(__linux__)
+    struct timespec t;
+    int ret = clock_gettime(CLOCK_BOOTTIME, &t);
```

Comment:
应该是CLOCK_MONOTONIC吧，BOOTTIME是include time the system was suspended.的

bpf_ktime_get_ns
Return the time elapsed since system boot, in nanoseconds.
Does not include time the system was suspended.
See: **clock_gettime**\ (**CLOCK_MONOTONIC**)
Returns Current *ktime*.